pkgs/tools/networking/davix/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit, cmake, pkgconfig, openssl, libxml2, boost }:
+
+stdenv.mkDerivation rec {
+  name = "davix-0.4.0";
+  buildInputs = [ stdenv pkgconfig cmake openssl libxml2 boost ];
+
+  src = fetchgit {
```

Comment:
Use fetchFromGitHub instead with the tag as `rev`. This avoids cloning the whole repository.